docs/docs/Tutorials/Simulation/SimulatorParameters.md

```diff
@@ -1,9 +1,119 @@
+- **Battery EMF**:  As an alternative to specifying the battery state of charge as a percentage, the zero-load battery voltage can be specfied, between 270V (empty battery), and 320V (full battery).
+- **Scale Factor**:  The buoy (and simulator) implements a default control algoritm in which the current in the motor windings is set as a function of motor RPM, with the resulting torque opposing the rotors motion.  The approximately implements a linear damping behavior and the specified Scale Factor is multiplied by the default Winding-Current/RPM relationship before being applied.  Allowable values are from 0.5 to 1.4, and the result is a simple way to change the damping the power-takeoff device is applying to the system.  As discussed in a later tutorial, this relationship can be over-ridden with external code, so this Scale Factor only applies to the default behavior of the system.
+
+
+#### Example Batch File
+The above parameters are spectified in a .yaml file that the batch-run tool reads in before execution begins.  A commented example is below and illustrates the use of the above parameters.  Lines that begin with # are comments and have no impact.
+
+```
+#
+# Batch-Specific Scalar Parameters
+#
+duration: 300
+seed: 42
+physics_rtf: 11
+#
+# Run-Specific Parameters (Test Matrix)
+#
+physics_step: [0.001, 0.01, 0.1]
+door_state: ['closed', 'open']
+scale_factor: [0.5, 0.75, 1.0, 1.3, 1.4]
+# May specify vector/scalar battery_soc (0.0 to 1.0) or battery_emf (270V to 320V)
+battery_soc: [0.25, 0.5, 0.75, 1.0]
+# battery_emf: [282.5, 295.0, 307.5, 320.0]
+IncidentWaveSpectrumType:
+ - MonoChromatic:
+     # A & T may be vector/scalar in pairs (A & T same length)
+     A: [1.0, 2.0, 3.0]
+     T: [12.0, 14.0, 15.0]
+ - Bretschneider:
+     # Hs & Tp may be vector/scalar in pairs (Hs & Tp same length)
+     Hs: 3.0
+     Tp: 14.0
+ # Multiple Custom Spectra must be listed individually (w & Szz are already vectors of same size)
+ - Custom:
+     w: [0.0, 0.2, 0.4, 0.6, 2.0]
+     Szz: [0.0, 0.4, 1.0, 1.0, 0.0]
+```
+
+As seen in this example, some parameters are enforced to be scalar values and apply to the entire batch of specified runs.  These are the specification of simulation duration (300), random seed (42), and the physics real-time factor (11). 
+
+The remaining run-specific parameters can be specified as arrays, and the batch-run tool then executes simulations for all possible combinations of these values.  Note that some values are specified in pairs.  For instance, three mono-chromatic waves are specified by this file with a specification of (A=1.0m, T=12s), (A=2.0m, T=14s), and (A=3.0m, T=15s), not nine runs that include all possible combinations of the specified amplitude and periods.
+
+Obviously it would be very easy to write a batch file specification that incluces thousands of runs, more practical usage will most likely iterate over a small number of parameters at at a time. 
```

Comment:
```suggestion
Obviously, it would be very easy to write a batch file specification that includes thousands of runs, but more practical usage will most likely iterate over a small number of parameters at at a time. 
```